grapher/controls/EntitySelectorModal.tsx

```diff
@@ -112,13 +112,19 @@ export class EntitySelectorModal extends React.Component<{
         document.removeEventListener("click", this.onDocumentClick)
     }
 
+    @action.bound onEscapeKeyDown(
```

Comment:
I am nitpicking, but should we maybe name this differently? 

In `onSearchKeyDown`, I think the "search" refers to the search field, which is the element we are attaching the event handler to. Here we are attaching the event handler to the overlay so maybe `onOverlayKeyDown`?

Because we have an if statement to check if the key is an escape, it doesn't seem like it's exactly `onEscapeKeyDown`.

Reply:
Of course! I just put in the name as a placeholder but the new name definitely makes more sense.